Plugins/Wox.Plugin.Folder/Main.cs

```diff
@@ -59,6 +65,26 @@ public List<Result> Query(Query query)
             return results;
         }
 
+        private static bool IsDriveOrSharedFolder(string search)
+        {
+            if (search.StartsWith(@"\\"))
+            { // share folder
+                return true;
+            }
+
+            if (_driverNames != null && _driverNames.Any(search.StartsWith))
```

Comment:
search.StartsWith - is this correct with no value provided?

Reply:
This is the same code as before, not sure what you mean by no value.
like -> search = " " ?
or -> search = "c" ?
or when you change the action keyword? and then get an empty value? but even so how is that an issue?

Reply:
all good never mind. Didnt know linq's expression _driverNames.Any(search.StartsWith) is same as _driverNames.Any(x => search.StartsWith(x)). Thats why i was wondering why StartsWith has no parameter passed.